docs/user-defined-ls/vscode-eslint-language-server.md

```diff
@@ -13,15 +13,15 @@ To enable [ESLint](https://eslint.org/) language support in your IDE, you can in
 
    ![Open file](../images/user-defined-ls/vscode-eslint-language-server/open_file.png)
 
-3. This will open the [New Language Server Dialog](../UserDefinedLanguageServer.md#new-language-server-dialog) with `Ada Language Server` pre-selected:
+3. This will open the [New Language Server Dialog](../UserDefinedLanguageServer.md#new-language-server-dialog) with `ESLint Language Server` pre-selected:
```

Comment:
Looks like there was still some copy/paste here that I've correct. Note that I've _only_ corrected these copy/paste references to "Ada" in the "ESLint" doc. I didn't actually review/audit the entire doc for accuracy.

Reply:
Thanks!